src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -1,213 +1,347 @@
   });
 
-  it('should not submit empty form', () => {
-    const onCalendarChange = sinon.spy();
-    const routeToNextAppointmentPage = sinon.spy();
-
-    const form = mount(
-      <DateTimeRequestPage
-        onCalendarChange={onCalendarChange}
-        data={{ calendarData: {} }}
-        routeToNextAppointmentPage={routeToNextAppointmentPage}
-      />,
-    );
-
-    form
-      .find('FormButtons')
-      .find('button[type="submit"]')
-      .simulate('click');
-    expect(routeToNextAppointmentPage.called).to.be.false;
-    form.unmount();
-  });
+  describe('Add community care appointment to calendar', async () => {
+    it('should allow user to request date and time for a community care appointment', async () => {
+      const routeToNextAppointmentPage = sinon.spy();
+      const routeToPreviousAppointmentPage = sinon.spy();
 
-  it('should not submit form with validation error', () => {
-    const onCalendarChange = sinon.spy();
-    const routeToNextAppointmentPage = sinon.spy();
-    const thirtyDaysFromNow = moment()
-      .add(30, 'day')
-      .format('YYYY-MM-DD');
-
-    const data = {
-      calendarData: { currentlySelectedDate: thirtyDaysFromNow, error: 'test' },
-    };
-
-    const form = mount(
-      <DateTimeRequestPage
-        onCalendarChange={onCalendarChange}
-        data={data}
-        routeToNextAppointmentPage={routeToNextAppointmentPage}
-      />,
-    );
-
-    form
-      .find('FormButtons')
-      .find('button[type="submit"]')
-      .simulate('click');
-    expect(routeToNextAppointmentPage.called).to.be.false;
-    expect(form.instance().isMaxSelectionsError()).to.be.false;
-    form.unmount();
-  });
+      // NOTE: Next available date is atleast 5 days from the current date
+      const selectedDate1 = getMondayTruFriday(moment().add(5, 'd')).format(
+        'YYYY-MM-DD',
+      );
+      const selectedDate2 = getMondayTruFriday(moment().add(6, 'd')).format(
+        'YYYY-MM-DD',
+      );
 
-  it('should have userSelectedSlot return false if no date selected', () => {
-    const onCalendarChange = sinon.spy();
-    const routeToNextAppointmentPage = sinon.spy();
-    const data = {
-      calendarData: {
-        currentlySelectedDate: '2020-12-20',
-        selectedDates: [],
-      },
-    };
-
-    const form = mount(
-      <DateTimeRequestPage
-        onCalendarChange={onCalendarChange}
-        data={data}
-        routeToNextAppointmentPage={routeToNextAppointmentPage}
-      />,
-    );
-
-    expect(form.instance().userSelectedSlot(data.calendarData)).to.be.false;
-    form.unmount();
-  });
+      // Seeding calendar with currently selected date and 2 previously selected dates
+      const screen = renderWithStoreAndRouter(
+        <DateTimeRequestPage
+          data={{
+            calendarData: {
+              currentlySelectedDate: selectedDate1,
+              selectedDates: [
+                { date: selectedDate1, optionTime: 'AM' },
```

Comment:
Once you get this test updated to use the page while connected to Redux, you could have your test click on these dates to select them, rather than setting them in the data directly.